src/runtime/arc.ts

```diff
@@ -583,6 +584,8 @@ ${this.activeRecipe.toString()}`;
         recipeHandle.storageKey = newStore.storageKey;
         continue;
         // TODO: move the call to ParticleExecutionHost's DefineHandle to here
+      } else if (recipeHandle.fate === 'use' && isInflatable) {
```

Comment:
'use' may refer only to handles that are in the arc (not in the context), and these should already be inflated, aren't they?

Reply:
It doesn't when the handle is based on a data resource in the manifest. I'll add a unit test showing this (which I should have done in the first place).

Reply:
i think that 'use' fate can only be used for stores already in the arc (not in the manifest). 
we had some tests that allowed it before, but it was by mistake, not by design.